bam_plcmd.c

```diff
@@ -1407,26 +1113,8 @@ int bam_mpileup(int argc, char *argv[])
         case 'q': mplp.min_mq = atoi(optarg); break;
         case 'Q': mplp.min_baseQ = atoi(optarg); break;
         case 'b': file_list = optarg; break;
-        case 'o': {
-                char *end;
-                long value = strtol(optarg, &end, 10);
-                // Distinguish between -o INT and -o FILE (a bit of a hack!)
-                if (*end == '\0') {
-                    mplp.openQ = value;
-                    fprintf(stderr, "[warning] samtools mpileup option "
-                            "'--open-prob INT' is functional, but deprecated. "
-                            "Please switch to using bcftools mpileup in future.\n");
-                } else {
-                    mplp.output_fname = optarg;
-                }
-            }
-            break;
-        case 'e': mplp.extQ = atoi(optarg); deprecated(c); break;
-        case 'h': mplp.tandemQ = atoi(optarg); deprecated(c); break;
+        case 'o': mplp.output_fname = optarg; break;
```

Comment:
There's a paragraph of “distinguishing -o” documentation to delete from _samtools-mpileup.1_ too (in the description of `--output`).